src/main/java/sirius/biz/analytics/metrics/DailyMetricComputer.java

```diff
@@ -50,36 +48,28 @@ public final void compute(LocalDate date, E entity, boolean bestEffort) throws E
 
         // if the reference date passed to this method is today, we consider the computation to be of particular
         // interest – unless it is only a best-effort computation that produces preliminary values only
```

Comment:
Update the comments about best-effort as well
```suggestion
        // if the reference date passed to this method is today, we consider the computation to be of particular
        // interest
```